content/docs/user-guide/visualizing-plots.md

```diff
@@ -217,11 +217,11 @@ details.
 [full format specification]:
   /doc/user-guide/project-structure/dvcyaml-files#top-level-plot-definitions
 
-### Plot Outputs
+### Plot outputs
```

Comment:
Unrelated but should we call this section "Stage-level plots" or just add the word (stage) in there? E.g.

```suggestion
### Plot outputs (stage-level)
```